src/components/forms/ActorSelect.tsx

```diff
@@ -90,7 +91,7 @@ export const ActorSelect = ({
           };
         }),
       ]);
-    } else if (sceneActorIds) {
+    } else if (context === "entity" && sceneActorIds) {
```

Comment:
I guess not a big issue because it's already not working, but it's unfortunate it has to be this way as one of the main uses of custom scripts is to configure actor behavior on button press (mostly the player tho...). Prefabs will be a good replacement for this anyway,

Reply:
Yeah it is a shame, I wonder if in the future maybe we can have the timer/input/music scripts have a small amount of memory for storing args, that would allow all the script parameters (both variables and actors) to be used as normal

Something like this
```
        VM_CONTEXT_PREPARE      1, ___bank_script_input_0, _script_input_0
        VM_PUSH_CONST           5 ; Actor Param
        VM_PUSH_CONST           VAR_HEALTH ; Variable Param
        VM_INPUT_ATTACH         128, ^/(1 | .OVERRIDE_DEFAULT)/, 2 ; <-- Take 2 params off stack
```
then within the input script itself we could read the values the same way we do in custom scripts today